package.json

```diff
@@ -28,17 +29,23 @@
     "postversion": "git push --follow-tags",
     "tdd": "yarn run test:mocha --watch",
     "test": "yarn run todos && yarn run lint && yarn run typecheck && yarn run test:mocha",
-    "test:mocha": "yarn run test:mocha:glob 'test/**/*test.js'",
-    "test:mocha:glob": "NODE_ENV=test mocha --opts mocha.opts",
+    "test:mocha": "yarn run test:mocha:glob 'test/**/*test.{js,jsx,ts,tsx}'",
+    "test:mocha:glob": "NODE_ENV=test mocha --opts mocha.opts --require @babel/polyfill --require .register.ts --extension ts,js,jsx,tsx",
     "todos": "yarn run todos:glob '**/*.{js,jsx,ts,tsx}'",
-    "todos:glob": "leasot --exit-nicely --tags '\\$FlowFixMe' --ignore='node_modules/**','lib/**','flow-typed/**'",
-    "typecheck": "flow"
+    "todos:glob": "leasot --exit-nicely --ignore='node_modules/**','lib/**'",
+    "typecheck": "tsc"
   },
   "dependencies": {
+    "@babel/preset-typescript": "^7.12.7",
```

Comment:
Will need to change in doc too

Reply:
@pma9 ah, I see – in a backend app, all babel plugins must be prod dependencies because we use babel in prod for the REPL and occasional `babel-node`ing of scripts, but in a library or a frontend it would be a dev dep. The guide was originally for a backend app.